logging/src/main/java/tech/pegasys/teku/logging/LoggingConfigurator.java

```diff
@@ -77,25 +80,27 @@ public static synchronized void addLoggersProgrammatically(
 
   private static void addLoggers(final AbstractConfiguration configuration) {
 
-    if (DESTINATION == null) {
+    if (isUninitialized() || noProgrammaticLoggingRequired()) {
       return;
```

Comment:
Probably useful to have an info log here saying that custom logging config is being used.